buildrunner/docker/multiplatform_image_builder.py

```diff
@@ -213,7 +213,7 @@ def _stop_local_registry(self):
             logger.warning("Local registry is not running when attempting to stop it")
 
     # pylint: disable=too-many-arguments
-    @retry(python_on_whales.exceptions.DockerException, tries=3, delay=5, logger=logger)
+    @retry(python_on_whales.exceptions.DockerException, tries=3, delay=10, logger=logger)
```

Comment:
Is there an exponential backoff retry available? That might work better than just straight up 10s delay each time?

Reply:
There is a backoff that I could set.
https://pypi.org/project/retry2/#:~:text=def%20retry(,True.%20default%3A%20None%0A%20%20%20%20%22%22%22